config/database-config.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  HOST: "dpg-cjadln5m2m9c73cf5rgg-a.singapore-postgres.render.com",
+  USER: "thetealab_user",
+  PASSWORD: "MbgbbcbYW27xlKl6v481X1XPnOX8Of7q",
+  DATABASE: "thetealab",
+  DIALECT: "postgres",
+  PORT: 5432,
+};

```

Comment:
Please remove this from here. This is a huge security issue. Consider using a .env file or add environment variables to github.